timber/src/main/java/timber/log/Timber.kt

```diff
@@ -22,7 +22,7 @@ class Timber private constructor() {
     internal val explicitTag = ThreadLocal<String>()
```

Comment:
There probably should be a change here as well, as you want users to be able to manually set tags